app/views/layouts/application.html.erb

```diff
@@ -8,6 +8,7 @@
     <%= csrf_meta_tags %>
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet"/>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,400i,600,600i,700,700i,800" rel="stylesheet">
```

Comment:
Why did we need this? Shouldn't this be coming from the css?

Reply:
It should...but it wasn't. The `@import url(...)` declaration showed up in the compiled CSS, but for some reason the browser was not making the network requests to Google fonts that it should. I went through the development tools for both my local instance and staging side by side to see why and the applied styles were identical, yet Chrome was rendering Helvetica Neue on the local instance. This was a real headache to debug, I tried updating Sass, tried to migrate modules forward, to no effect. Adding this line finally fixed it, but more investigation needs to be done about our Sass system (some parts of it are ancient) and webpack config.

Reply:
Yeah, this was my blocker. Let's add a comment and a Jira tasks to understand this. This seems like a side fix that is going to come back to haunt us.